README.md

```diff
@@ -1,4 +1,7 @@
 # Open Pension Scraper
+
+[![Build Status][travis-image]][travis-url] [![Join the chat at https://gitter.im/open-pension/Lobby][gitter-image]][gitter-url]
```

Comment:
You added both `travis` and `gitter`, keep only `travis`.

And this is just the urls for the image and the links. look at the [README](https://raw.githubusercontent.com/hasadna/open_pension/master/README.md) at the end of the file for the relevant links (they should be a little different, there are the links to open pension travis, we need our travis).